src/column_variables.jl

```diff
@@ -106,12 +128,20 @@ function reset_column!(column::ColumnVariables{NF}) where NF
     fill!(column.temp_tend,0)
     fill!(column.humid_tend,0)
 
+    # Convection
     column.cloud_top = column.nlev+1
-    column.precip_large_scale = zero(NF)
+    column.conditional_instability = false
+    column.activate_convection = false
```

Comment:
@milankl I decided to use these flags instead of the convention that `cloud_top == nlev+1` means no convection, as I think it's cleaner, safer, and more easier to understand!

Reply:
Nice, yeah!